### DISCUSSION TITLE
47US Web Admin Password

### DISCUSSION THREAD

**Discussion by noz1380** [alienatedsec/yi-hack-v5#258] [Q&A] [answered]
Hi,

I originally setup my camera for v4, accessed the web page and setup ftp and telnet.  Via telnet I changed the password so I could use the ftp and HA.

I didnt get this to work (new to HA) and found the v5 (thank you btw) and decided to try rtsp.

I have flashed the new fw, but cannot get access to the web page, as I dont know the username and password, I cannot turn on telnet (no access the web).  I have sucessfully ssh into the camera, but this was using the password i set via telnet when running v4.

I guess my question is, can I start from scratch, complete refresh?  A reset doesnt do this for me.

Alternativly can I set passwords via ssh as I do have access that way.

Sorry for being a noob.

**Comment by alienatedsec** [comment]
> I have flashed the new fw

Which version please?

**Reply by noz1380** [reply]
Y18 0.3.7

**Reply by alienatedsec** [reply, marked as answer]
Try #257

**Reply by noz1380** [reply]
Ok I'll give it a go. 

Is there a way to have a clean start??

**Reply by noz1380** [reply]
> Try #257

Think that has me sorted, tho I'm having issues with the rtsp, but I added that to another bug post. 

Thank you for your help!!

**Comment by phrogg** [comment]
Interestingly I found my old camera and seem to have the exact same problem. With the latest software.
